tinyiiod.h

```diff
@@ -44,7 +47,9 @@ struct tinyiiod_ops {
 	int32_t (*open)(const char *device, size_t sample_size, uint32_t mask);
 	int32_t (*close)(const char *device);
 
-	ssize_t (*read_data)(const char *device, char *buf, size_t bytes_count);
+	ssize_t (*read_data)(const char *device, char *pbuf, size_t bytes_count);
+	ssize_t (*write_data)(const char *device, const char *buf,
+			      size_t bytes_count);
```

Comment:
the `const char *` conversion is nice; it breaks the API a bit, but it's not too bad as people would get compiler warnings
still, i'd put it it's own commit

Reply:
write_data is something new here. It newer existed before. Please be more explicit.

Reply:
my bad here;
i thought it was modifying existing stuff